app/views/govuk_publishing_components/components/_autocomplete.html.erb

```diff
@@ -0,0 +1,42 @@
+<%
+add_gem_component_stylesheet("autocomplete")
+display_number_suggestions ||= 5
+suggestion_icon ||= ''
+request_debounce_time ||= ''
+source ||= nil
+position_results ||= nil
+submit_form_on_select ||= false
+idPostfix ||= "#{SecureRandom.hex(4)}"
```

Comment:
What is `idPostfix` used for? If this is an option that can be passed to the component, it should have an example and explanation in the YML file.

Reply:
no it cant be passed to component (so i will remove the ||). it is used to append to ids for elements within autocomplete so they are unique (in case another autocomplete is on page)